src/index.js

```diff
@@ -36,6 +36,7 @@ export {
 export { default as Menu, MenuItem, MenuList } from './Menu';
 export { default as Paper } from './Paper';
 export { CircularProgress, LinearProgress } from './Progress';
+export { default as Popover } from './internal/Popover';
```

Comment:
While we plan on exposing the Popover component, this change can't be accepted. We need some documentation and to move the component outside of the internal folder.